solidity/contracts/bridge/Bridge.sol

```diff
@@ -19,7 +19,6 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 
```

Comment:
What do you think about noting the ordering convention in the `@dev` section of the `Bridge` contract? It will make navigation easier for newcomers and let us remember the preferred order while introducing new functions.

Reply:
94a10240be3e21d912830eedc16ec7e58a606400